Bugsnag/include/Bugsnag/BugsnagStackframe.h

```diff
@@ -8,6 +8,12 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString * BugsnagStackframeType NS_TYPED_ENUM;
+
+FOUNDATION_EXPORT BugsnagStackframeType const BugsnagStackframeTypeCocoa;
```

Comment:
Would we also need enum values for react-native and other frameworks on top of ios?

Reply:
No - this exists only so that BugsnagReactNative can set the type of native stack frames. The reactnativejs type is applied to the containing error, which specifies the type for all frames that don't explicitly state a type.